### ISSUE TITLE
[Bug] API error: "PersistedQueryNotFound"

### ISSUE THREAD

**Issue by jaredcat** [TychoTheTaco/Twitch-Drops-Bot#287]
```
[error] Failed to get drop campaign details
[info] Processing campaign: Halo Infinite HCS Kickoff Major Charlotte Co-Streamers - FEB 24-26
[error] Error processing campaign! Error: API error: {
    "errors": [
        {
            "message": "PersistedQueryNotFound"
        }
    ],
    "extensions": {
        "durationMilliseconds": 3,
        "operationName": "DropCampaignDetails",
        "requestID": "01GT2829C86ER2W5HZ03ENVKET"
    }
}
[info] Processing campaign: Halo Infinite HCS Kickoff Major Charlotte FEB 24-26
[info] Found 2 good streams
[info] Watching stream: https://www.twitch.tv/halo
```

![image](https://user-images.githubusercontent.com/3268576/221250662-31651865-d32e-4cec-8592-174f09451336.png)

**Comment by HoldOnBro** [comment]
same, guess Twitch modded their api aaaaaaaagain

**Comment by BulldozerPete** [comment]
Looks like this project is dead. Last change was on 3rd of Dec.

Did anyone made a fork and fixed that bug?

**Comment by FlickF** [comment]
To fix the issue you have to compile the project with pull request #292.

**Comment by BulldozerPete** [comment]
I don't know how to compile. I am using the docker container.

**Comment by atornatore** [comment]
> To fix the issue you have to compile the project with pull request #292.

For me it didn't solve the problem but maybe I did something wrong.
Please @TychoTheTaco, review suggested PR #292

**Comment by gromez** [comment]
> > To fix the issue you have to compile the project with pull request #292.
> 
> For me it didn't solve the problem but maybe I did something wrong. Please @TychoTheTaco, review suggested PR #292

I also have a version with some updated sha265 sums but autoClaim is not working : `API error: failed integrity check`
Script is detected as bad bot, it must be why.

**Comment by BulldozerPete** [comment]
Last activity of @TychoTheTaco was on 17th of Dec 2022.

**Comment by deathart** [comment]
If you wish I can make you a public docker image

**Comment by gromez** [comment]
> If you wish I can make you a public docker image

autoClaim is working on your version?

**Comment by deathart** [comment]
No the auto claim of drop or point does not work I did not have time to look, but the rest works perfectly

**Comment by jaredcat** [comment]
> autoClaim is working on your version?

I just use TamperMonkey chrome extension (and a [script like this one](https://greasyfork.org/en/scripts/439734-twitch-auto-claim-drop)) and leave a browser tab open on [the drops page to claim the drops](https://www.twitch.tv/drops/inventory). If not that, at least you're getting some drops that you can claim manually, and the bot will continue once you do.

> Looks like this project is dead. Last change was on 3rd of Dec.

That's actually not that long of time in open-source world, but I'm guessing the maintainer is discouraged from working on it because of Twitch's anit-bot measure that broke claiming

**Comment by jaredcat** [comment]
I forked @deathart fixes and fixed the error being thrown by tags: https://github.com/TychoTheTaco/Twitch-Drops-Bot/pull/293

Docker image: `ghcr.io/updownleftdie/twitch-drops-bot:latest`

**Comment by jaredcat** [comment]
If anyone knows how this project gets around the integrity check, I can try fixing channel points and maybe claiming drops: https://github.com/rdavydov/Twitch-Channel-Points-Miner-v2

**Comment by HoldOnBro** [comment]
> If anyone knows how this project gets around the integrity check, I can try fixing channel points and maybe claiming drops: https://github.com/rdavydov/Twitch-Channel-Points-Miner-v2

hope copilotx would know it

**Comment by TychoTheTaco** [comment]
> > Looks like this project is dead. Last change was on 3rd of Dec.
> 
> That's actually not that long of time in open-source world, but I'm guessing the maintainer is discouraged from working on it because of Twitch's anit-bot measure that broke claiming

Yea that's the main reason I stopped working on it. I would like to at least keep the auto-watching working so I merged #292 which fixes this issue
